tests/cache.spec.coffee

```diff
@@ -20,27 +20,27 @@ describe 'Cache:', ->
 		describe 'given a cache directory', ->
 
 			beforeEach ->
-				@resinSettingsGetStub = m.sinon.stub(resin.settings, 'get')
+				@balenaSettingsGetStub = m.sinon.stub(balena.settings, 'get')
 
 				if os.platform() is 'win32'
-					@resinSettingsGetStub
+					@balenaSettingsGetStub
 						.withArgs('cacheDirectory')
-						.returns(Promise.resolve('C:\\Users\\johndoe\\_resin\\cache'))
+						.returns(Promise.resolve('C:\\Users\\johndoe\\_balena\\cache'))
 				else
-					@resinSettingsGetStub
+					@balenaSettingsGetStub
 						.withArgs('cacheDirectory')
-						.returns(Promise.resolve('/Users/johndoe/.resin/cache'))
+						.returns(Promise.resolve('/Users/johndoe/.balena/cache'))
 
 			afterEach ->
-				@resinSettingsGetStub.restore()
+				@balenaSettingsGetStub.restore()
 
 			describe 'given valid slugs', ->
 
 				beforeEach ->
-					@getManifestBySlugStub = m.sinon.stub(resin.models.device, 'getManifestBySlug')
+					@getManifestBySlugStub = m.sinon.stub(balena.models.device, 'getManifestBySlug')
 					@getManifestBySlugStub.withArgs('raspberry-pi').returns Promise.resolve
 						yocto:
-							fstype: 'resin-sdcard'
+							fstype: 'balena-sdcard'
```

Comment:
Not sure this is gonna change yet, should check with the OS team

Reply:
True - reverted.

It doesn't actually matter here, the code just tests `!= 'zip`, but more accurate test data is definitely better anyway.